tests/keycloak/import/ehrbase-realm-exported.json

```diff
@@ -0,0 +1,2143 @@
+  "id" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+  "realm" : "ehrbase",
+  "displayName" : "EHRBase",
+  "displayNameHtml" : "",
+  "notBefore" : 0,
+  "defaultSignatureAlgorithm" : "RS256",
+  "revokeRefreshToken" : false,
+  "refreshTokenMaxReuse" : 0,
+  "accessTokenLifespan" : 300,
+  "accessTokenLifespanForImplicitFlow" : 900,
+  "ssoSessionIdleTimeout" : 1800,
+  "ssoSessionMaxLifespan" : 36000,
+  "ssoSessionIdleTimeoutRememberMe" : 0,
+  "ssoSessionMaxLifespanRememberMe" : 0,
+  "offlineSessionIdleTimeout" : 2592000,
+  "offlineSessionMaxLifespanEnabled" : false,
+  "offlineSessionMaxLifespan" : 5184000,
+  "clientSessionIdleTimeout" : 0,
+  "clientSessionMaxLifespan" : 0,
+  "clientOfflineSessionIdleTimeout" : 0,
+  "clientOfflineSessionMaxLifespan" : 0,
+  "accessCodeLifespan" : 60,
+  "accessCodeLifespanUserAction" : 300,
+  "accessCodeLifespanLogin" : 1800,
+  "actionTokenGeneratedByAdminLifespan" : 43200,
+  "actionTokenGeneratedByUserLifespan" : 300,
+  "oauth2DeviceCodeLifespan" : 600,
+  "oauth2DevicePollingInterval" : 5,
+  "enabled" : true,
+  "sslRequired" : "none",
+  "registrationAllowed" : false,
+  "registrationEmailAsUsername" : false,
+  "rememberMe" : false,
+  "verifyEmail" : false,
+  "loginWithEmailAllowed" : true,
+  "duplicateEmailsAllowed" : false,
+  "resetPasswordAllowed" : true,
+  "editUsernameAllowed" : false,
+  "bruteForceProtected" : false,
+  "permanentLockout" : false,
+  "maxTemporaryLockouts" : 0,
+  "maxFailureWaitSeconds" : 900,
+  "minimumQuickLoginWaitSeconds" : 60,
+  "waitIncrementSeconds" : 60,
+  "quickLoginCheckMilliSeconds" : 1000,
+  "maxDeltaTimeSeconds" : 43200,
+  "failureFactor" : 30,
+  "roles" : {
+    "realm" : [ {
+      "id" : "19509040-3f55-488a-a520-ab22accf7a36",
+      "name" : "domain-expert",
+      "composite" : false,
+      "clientRole" : false,
+      "containerId" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+      "attributes" : { }
+    }, {
+      "id" : "de2ada1d-99b3-4de1-8c2d-6da1598ef2dd",
+      "name" : "uma_authorization",
+      "description" : "${role_uma_authorization}",
+      "composite" : false,
+      "clientRole" : false,
+      "containerId" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+      "attributes" : { }
+    }, {
+      "id" : "4eab895c-d58d-47cc-b208-89322ae9afed",
+      "name" : "USER",
+      "description" : "",
+      "composite" : false,
+      "clientRole" : false,
+      "containerId" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+      "attributes" : { }
+    }, {
+      "id" : "b1342bc8-0b6c-448f-8aaf-c81574f30595",
+      "name" : "compensation-admin",
+      "composite" : false,
+      "clientRole" : false,
+      "containerId" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+      "attributes" : { }
+    }, {
+      "id" : "e47d350d-a951-4ddb-9923-657b981ce38f",
+      "name" : "tenant-admin",
+      "composite" : false,
+      "clientRole" : false,
+      "containerId" : "ca3e0ff6-5282-4d18-b860-8d28d7ac1b86",
+      "attributes" : { }
+    }, {
```

Comment:
we should double check what roles are needed ;).

Reply:
@alexlehn removed all others roles + users and kept the needed only.